test/train/test_train_utils.py

```diff
@@ -0,0 +1,112 @@
+import unittest
+from fruit_classifier.train.train_utils import get_image_paths
+from fruit_classifier.train.train_utils import get_data_and_labels
+from fruit_classifier.train.train_utils import get_model_input
+from fruit_classifier.train.train_utils import get_model
+from fruit_classifier.train.train_utils import train_model
+from fruit_classifier.train.train_utils import plot_training
+from fruit_classifier.preprocessing.preprocessing_utils import \
+    get_image_generator
+from pathlib import Path
+import shutil
+
+
+class TestTrainUtils(unittest.TestCase):
+    def setUp(self):
```

Comment:
Recall that the `setUp` is common for all member functions.
If a variable is not shared with at least two member functions it should be refactored to the member function needing it.

For example:
Everything under
`# Run get_model_input`
is only needed in the `test_get_model_input` function, and should thus be moved there.